packages/driver/cypress/integration/e2e/multi-domain/multi_domain_cypress_api.spec.ts

```diff
@@ -0,0 +1,253 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests+
+describe('multi-domain Cypress API', { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
+  beforeEach(() => {
+    cy.visit('/fixtures/multi-domain.html')
+    cy.get('a[data-cy="multi-domain-secondary-link"]').click()
+  })
+
+  context('Commands', () => {
+    context('add', () => {
+      it('adds a custom command', () => {
+        cy.switchToDomain('foobar.com', () => {
+          // @ts-ignore
+          Cypress.Commands.add('foo', () => {
+            return cy.wrap('bar')
```

Comment:
```suggestion
            cy.wrap('bar')
```

Never need to return cypress commands. This could also just `return 'bar'`, there's no need to wrap it. Same for others in this spec.

Reply:
this and the lot addressed in ddf1daf3bf. Also removed the wrapping and went for returning the raw string in d352729d88